src/components/navbars/DesktopNav.jsx

```diff
@@ -0,0 +1,35 @@
+import loginIcon from '../../assets/icons/login.png';
+import searchIcon from '../../assets/icons/search.png';
+
+const DesktopNav = ({ onSearch, onLogin, onJoin, stateSearch, stateLogin, stateJoin }) => {
+    return (
+        <nav className="w-2/3 hidden relative py-6 sm:flex justify-between">
+            {
+                stateSearch
+                    ? <form className="border-2 border-red rounded-full h-10 w-2/4 flex justify-between items-center px-4 ml-6 gap-1">
```

Comment:
The search bar would benefit from being its own component if the intention is to render it throughout the app.

Reply:
I think that the value of this search bar will be passed to the search bar on the search page when the search page will be ready. If not, then I will move the search into a separate component and it will look different depending on the page. So it could be done later, I suppose.